pkgs/tools/backup/borgmatic/default.nix

```diff
@@ -33,12 +33,11 @@ python3Packages.buildPythonApplication rec {
       --bash <($out/bin/borgmatic --bash-completion)
 
     mkdir -p $out/lib/systemd/system
-    cp sample/systemd/borgmatic.timer $out/lib/systemd/system/
```

Comment:
Wouldn't this remove the timer completely and breaking the module?

Reply:
As I understand, using startAt and timer are equivalent.

```
systemd.services.<name>.startAt

Automatically start this unit at the given date/time, which must be in the format described in [systemd.time(7)](https://www.freedesktop.org/software/systemd/man/systemd.time.html). This is equivalent to adding a corresponding timer unit with OnCalendar set to the value given here.
```

For example, https://github.com/NixOS/nixpkgs/blob/nixos-22.11/nixos/modules/services/misc/gitlab.nix#L1491

Reply:
Yep. I tested this on my machine. It's also worth noting that `sample/systemd/borgmatic.timer` is intended to be a sample/example systemd file, so it's ok if we use a more nix-y `systemd.services.borgmatic.startAt` instead.